Meros.nimble

```diff
@@ -14,7 +14,7 @@ skipExt = @["nim"]
 
 #Dependencies
 requires "nim >= 1.0.4"
-requires "https://github.com/MerosCrypto/Argon2"
+requires "https://github.com/MerosCrypto/Argon2 >= 1.1.2"
```

Comment:
Why was this needed? I know we have the same thing for ForceCheck, yet I honestly forget why.

Reply:
It's because `nimble` cache might have a previous incompatible version installed and would accurately satisfy the dependency. This was true here for `Argon2`. By setting a version, it then forces that version to be updated (automatically if I remember correctly) in the case that you have a previous version installed.

We should be doing this for all dependencies to be honest but didn't want to tie it in with the other changes here.

Reply:
Thanks for the clarification. Since we're already doing it with Argon2, feel free to. Else, we can create a new thing immediately after this is merged.